app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+
+
+def copy_file(command: str) -> None:
+    names = command.split()
+    if names[1] == names[2] or\
+            len(command.split()) != 3 or\
+            command.split()[0] != "cp":
+        return
+    with open(names[1], "r") as f_r, open(names[2], "w") as f_w:
```

Comment:
`f_r` and `f_w` are not descriptive variable names

Reply:
Done